brainstormforce/ultimate-addons-for-gutenberg
# Improvement: Updated copy paste popup designs as per suggestion

### Description
1. Copy - Paste style popup stays open even after we click on the outside.
2. Added icons to copy and paste style popup.
3. Added border to popup.

### Types of changes
non-breaking change which fixes an issue

### Test Cases
1. Drag and drop any spectra block 
2. Click on the copy/paste icon on block controls
3. Then click outside of the popover
4. Check popup closed or not

### Checklist:
